llmfoundry/models/hf/hf_causal_lm.py

```diff
@@ -24,6 +23,12 @@
 from llmfoundry.models.hf.model_wrapper import HuggingFaceModelWithZLoss
 from llmfoundry.models.utils import init_empty_weights
 
+try:
+    from peft.peft_model import PeftModel
+    model_types = (PeftModel, transformers.PreTrainedModel)
+else:
```

Comment:
```suggestion
except ImportError:
```

Reply:
wanted to push before a mtg.
that was hilarious 🤣